keyboards/xd75/keymaps/davidrambo/keymap.c

```diff
@@ -1,37 +1,56 @@
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
 #include QMK_KEYBOARD_H
 
 //aliases for clarity in layering
-#define A_BSPC  LALT(KC_BSPC) // delete whole word in Mac
-#define C_BSPC  LCTL(KC_BSPC) // delete whole word in PC
-#define A_LEFT  LALT(KC_LEFT)
-#define A_RGHT  LALT(KC_RGHT)
-#define C_RGHT  LCTL(KC_RGHT)
-#define C_LEFT  LCTL(KC_LEFT)
-#define SftEnt  SFT_T(KC_ENT)
-#define BS_GUI  LGUI_T(KC_BSPC)
-#define BS_CTL  LCTL_T(KC_BSPC)
+#define ABSPC  LALT(KC_BSPC) // delete whole word in Mac
+#define CBSPC  LCTL(KC_BSPC) // delete whole word in PC
+#define ALEFT  LALT(KC_LEFT)
+#define ARGHT  LALT(KC_RGHT)
+#define CRGHT  LCTL(KC_RGHT)
+#define CLEFT  LCTL(KC_LEFT)
+#define SftEnt SFT_T(KC_ENT)
+#define BSGUI  LGUI_T(KC_BSPC)
+#define BSCTL  LCTL_T(KC_BSPC)
 
 // internet browser tab shortcuts and window/application swapping for Mac and Win
 #define GSL     LGUI(S(KC_LEFT)) // back one tab in Safari
 #define GSR     LGUI(S(KC_RGHT)) // forward one tab in Safari
-#define CTLPGDN LCTL(KC_PGDN)   // back one tab on Windows
-#define CTLPGUP LCTL(KC_PGUP)   // forward one tab on Windows
-#define G_TAB   LGUI(KC_TAB)    // Mac: switch applications
-#define G_GRV   LGUI(KC_GRV)    // Mac: switch between windows within an application
-#define A_TAB   LALT(KC_TAB)
-#define C_TAB   LCTL(KC_TAB)
-
-// navigation layers for both Mac OS X and Windows
-#define NAV     LT(4, KC_TAB)
-#define NAVPC   LT(5, KC_TAB)
+#define CTLPGDN LCTL(KC_PGDN)   // back one tab on PC
+#define CTLPGUP LCTL(KC_PGUP)   // forward one tab on PC
+#define GTAB    LGUI(KC_TAB)    // Mac: switch applications
+#define GGRV    LGUI(KC_GRV)    // Mac: switch between windows within an application
+#define ATAB    LALT(KC_TAB)
+#define CTAB    LCTL(KC_TAB)
+
+// navigation layers for both Mac OS X and PC
+#define NAV     LT(5, KC_ESC)
+#define NAVPC   LT(6, KC_ESC)
+#define NAVQUD  MO(7)
 
 // Layer shorthand
 #define _COLEMAK 0
 #define _PC 1
 #define _GAME 2     // Gaming layer
-#define _SYMBOL 3   // Function keys, numbers, symbols, Backlighting
-#define _NAV 4   // Navigation Layer on Mac
-#define _NAVPC 5 // Navigation Layer on Win
+#define _QUD  3     // Roguelike layer
+#define _SYMBOL 4   // Function keys, numbers, symbols, Backlighting
+#define _NAV 5      // Navigation Layer on Mac
+#define _NAVPC 6    // Navigation Layer on Win
+#define _NAVQUD 7   // Numpad for Roguelike 8-directional movement
```

Comment:
```suggestion
enum custom_layer {
    _COLEMAK,
    _PC,
    _GAME,     // Gaming layer
    _QUD,      // Roguelike layer
    _SYMBOL,   // Function keys, numbers, symbols, Backlighting
    _NAV,      // Navigation Layer on Mac
    _NAVPC,    // Navigation Layer on Win
    _NAVQUD,   // Numpad for Roguelike 8-directional movement
};
```

Reply:
Oh gosh, I should have inferred from your previous suggested change that I'd have similar changes to make on my other keymaps! Thank you, drashna.